stdlib/builtin/thread.rbs

```diff
@@ -1038,10 +1038,8 @@ class Thread::Queue < Object
   # Also aliased as: [size](Queue.downloaded.ruby_doc#method-i-size)
   def length: () -> Integer
 
-  def marshal_dump: () -> untyped
```

Comment:
I can't find any documentation for `Thread::Queue#marshal_dump`

https://ruby-doc.org/core-2.7.0/Queue.html